modules/installation-osp-kuryr-known-limitations.adoc

```diff
@@ -11,11 +11,13 @@ Using {product-title} with Kuryr SDN has several known limitations.
 [id="openstack-general-limitations_{context}"]
 == {rh-openstack} general limitations
 
-{product-title} with Kuryr SDN does not support `Service` objects with type `NodePort`.
+Using {product-title} with Kuryr SDN has several limitations that apply to all versions and environments:
 
-If the machines subnet is not connected to a router, or if the
-subnet is connected, but the router has no external gateway set,
-Kuryr cannot create floating IPs for `Service` objects with type `LoadBalancer`.
+* `Service` objects with the `NodePort` type are not supported.
+
+* Clusters that use the OVN Octavia provider driver support `Service` objects for which the `.spec.selector` property is unspecified only if the `.subsets.addresses` property of the `Endpoints` object includes the nodes' or pods' subnet. 
```

Comment:
To avoid possessive apostrophes here, I would suggest writing, "...includes the subnet of the nodes or pods" or "includes the node or pod subnet". (ISG mentions this on p. 41.)